qiskit/algorithms/gradients/utils.py

```diff
@@ -377,3 +379,103 @@ def _gate_gradient(gate: Gate) -> Instruction:
         czx = czx_circ.to_instruction()
         return czx
     raise TypeError(f"Unrecognized parameterized gate, {gate}")
+
+
+def _make_lin_comb_qfi_circuit(
+    circuit: QuantumCircuit, add_measurement: bool = False
+) -> dict[Parameter, list[LinearCombGradientCircuit]]:
+    """Makes gradient circuits for the linear combination of unitaries method.
+
+    Args:
+        circuit: The original quantum circuit.
+        add_measurement: If True, add measurements to the gradient circuit. Defaults to False.
+            ``LinCombSamplerGradient`` calls this method with `add_measurement` is True.
+
+    Returns:
+        A dictionary mapping a parameter to the corresponding list of ``LinearCombGradientCircuit``
+    """
+    supported_gates = [
+        "rx",
+        "ry",
+        "rz",
+        "rzx",
+        "rzz",
+        "ryy",
+        "rxx",
+        "cx",
+        "cy",
+        "cz",
+        "ccx",
+        "swap",
+        "iswap",
+        "h",
+        "t",
+        "s",
+        "sdg",
+        "x",
+        "y",
+        "z",
+    ]
+
+    circuit2 = transpile(circuit, basis_gates=supported_gates, optimization_level=0)
```

Comment:
An Estimator receiving this circuit presumably will further transpile this circuit. I assume that is all fine - testing via the reference estimator, which uses quantum info, I don't think has such paths right where it  would do further transpilation.